src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -2138,17 +2141,46 @@ void Reduce::reduce_PLN(const uint8_t *in_ptr, uint8_t *out_ptr) {
                                           IW - tail_start, 0, IH);
                 });
             } else if (!ReduceC && ReduceD && ReduceH && !ReduceW) {
-                parallel_for(IC, [&](size_t ic) {
-                    size_t oc = ic; GET_PTR_NC_PLN;
-                    parallel_for(IW / blk_size, [&](size_t ibw){
-                        size_t obw = ibw;
-                        reduce_kernel_process(in_ptr_nc + ibw * blk_size * src_data_size, out_ptr_nc + obw * blk_size * dst_data_size,
-                                              blk_size, 0, ID * IH);
+                size_t IWB = IW / blk_size;
+                if (IC == 1 && ID > 1) {
+                    // reduce parallelly in D dimension
+                    // step1: !ReduceD && ReduceH && !ReduceW
+                    uint8_t *prc_ptr_n = static_cast<uint8_t *>(reduceDH_prc_mem.get());
+                    init_dst_data(prc_ptr_n, prc_size);
+                    parallel_for2d(ID, IWB, [&](size_t id, size_t iwb){
```

Comment:
I am just wondering about best approach here. 
Lets consider that task NWHC -> N11C. Threading by C seems to be not the best solution since this is innermost axis. More optimal solution might be to split reduction area [H*W] on several tiles of the same size and compute reduction of each tile independently in different thread (like reduction([H/block_size * W * C]) = [1 * 1 * C]). After that you can have small finalization step that will make reduction between the tiles. What do you think?
I don't insist to do this in bound of this PR. Mostly thoughts for the future.